.github/workflows/automatic-pr-update.yml

```
@@ -20,6 +10,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: rasahq/merge-pal-action@master
+      - uses: rasahq/update-pr-branch@601ac3dadf708b14e7f0252d98230538235f99d9
+        if: github.repository == 'RasaHQ/rasa'
         with:
           token: ${{ secrets.RASABOT_AUTOMERGE_GITHUB_TOKEN || secrets.GITHUB_TOKEN}}
+          # required parameter by original action - 
+          # check is already done through protected branches so not needed for us
+          required_approval_count: 0   
```

Just me being picky 😅

```suggestion
          required_approval_count: 0
```

---

very much appreciated :D - an obvious flaw of the in-browser editor